app/javascript/components/mappers/domainToApi/lease.js

```diff
@@ -7,10 +7,10 @@ export const leaseFieldMapper = {
   'no_preference_used': 'no_preference_used',
   'preference_used': 'preference_used',
   'unit': 'unit',
-  total_monthly_rent_without_parking: (source) => source.total_monthly_rent_without_parking,
```

Comment:
looking at this more it looks like we could maybe get rid of this completely? Do you thin that would break anything?

Reply:
removed and working